maintainers/upload-debug-info-to-sentry.py

```
@@ -1,5 +1,5 @@
 #!/usr/bin/env nix
-#!nix shell --inputs-from . nixpkgs#sentry-cli --command python3
+#!nix shell --inputs-from . nixpkgs#sentry-cli nixpkgs#python3 nixpkgs#binutils --command python3
```

Ruff (0.15.10)

[warning] Shebang should be at the beginning of the file

(EXE005)